NOTES-Valgrind.md

```diff
@@ -17,21 +17,21 @@ Requirements
    See http://valgrind.org/info/platforms.html
 2. Valgrind installed on the platform
    See http://valgrind.org/downloads/current.html
-3. OpensSSL compiled, sSee [INSTALL.md](INSTALL.md)
+3. OpensSSL compiled, see [INSTALL.md](INSTALL.md)
 
 Running Tests
 -------------
 
 Test behavior can be modified by adjusting environment variables.
 
-    EXE_SHELL
+`EXE_SHELL`
 
 This variable is used to specify the shell used to execute OpenSSL test
 programs. The default wrapper (`util/wrap.pl`) initializes the environment
 to allow programs to find shared libraries. The variable can be modified
 to specify a different executable environment.
 
-    EXE_SHELL="`/bin/pwd`/util/wrap.pl valgrind --error-exitcode=1 --leak-check=full -q"
+`EXE_SHELL="`/bin/pwd`/util/wrap.pl valgrind --error-exitcode=1 --leak-check=full -q"`
```

Comment:
You might want to change `` `/bin/pwd` `` to `$(/bin/pwd)`, or double the outermost backticks.  See https://spec.commonmark.org/0.29/#code-spans

Reply:
> You might want to change `/bin/pwd` to $(/bin/pwd), or double the outermost backticks.

Thx for the hint, but not needed.
Due to the 4 leading spaces this is correctly recognized as code block.

Reply:
So you went back to ugly?

Reply:
> So you went back to ugly?

I put back the four spaces and the result looks good, at least to me, see
https://github.com/siemens/openssl/blob/rename-NOTES-etc/NOTES-Valgrind.md#running-tests

Reply:
And if you read it with `less`?

Reply:
I also see no issue with the `less` output:
```
This variable is used to specify the shell used to execute OpenSSL test
programs. The default wrapper (`util/wrap.pl`) initializes the environment
to allow programs to find shared libraries. The variable can be modified
to specify a different executable environment.

    EXE_SHELL="`/bin/pwd`/util/wrap.pl valgrind --error-exitcode=1 \
        --leak-check=full -q"

```